Add support for Workflow Steps from Apps (#607)
slackapi/bolt-js: docs/_steps/executing_workflow_steps.md

title: Executing workflow steps
lang: en
slug: executing-steps
order: 5
---

<div class="section-content">

> When your workflow step is executed by an end user, your app will receive a `workflow_step_execute` event. The method assigned to the `execute` property of the `WorkflowStep` configuration object passed in during instantiation will run when this event occurs.

Reviewer: ```suggestion
When your workflow step is executed by an end user, your app will receive a [`workflow_step_execute` event](https://api.slack.com/events/workflow_step_execute). The callback assigned to the `execute` property of the `WorkflowStep` configuration object will run when this event occurs.
```

> Using the `inputs` from the configuration modal submission in the `save` callback, this is where we make third-party API calls, save things to a database, update the end user's Home Tab, and/or decide what outputs will be available to subsequent workflow steps by mapping values to the `outputs` object.

Reviewer: ```suggestion
Using the `inputs` from the `save` callback, this is where you can make third-party API calls, save information to a database, update the user's Home tab, or decide the outputs that will be available to subsequent workflow steps by mapping values to the `outputs` object.
```
